app/views/worldwide_offices/show.html.erb

```diff
@@ -8,7 +8,7 @@
     <% if headers.any? %>
       <nav>
         <div class="content">
-          <h1>Contents</h1>
+          <h1 lang="en">Contents</h1>
           <ol>
```

Comment:
We could use a heading component and a [contents list component](https://components.publishing.service.gov.uk/component-guide/contents_list) here.